assets/css/home.css

```diff
@@ -0,0 +1,712 @@
+  font-size: 30px;
+  font-weight: bold;
+  text-decoration: none;
+}
+
+.social-media {
+  display: none;
+}
+
+article.center-text {
+  text-align: center;
+}
+
+.set-background {
+  background-image: url(../images/bg/mobile_intro_bg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  padding-inline: 8vw;
+  margin: 2px 1px;
+}
+
+.Hero {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: calc(min(80px, 10%)) 0;
+  gap: 10px;
+}
+
+h2 {
+  margin-top: 10%;
+  margin-bottom: 0;
+  font-size: 1.6rem;
+  font-weight: 600;
+}
+
+h1 {
+  font-weight: 900;
+  font-size: 3rem;
+  text-transform: uppercase;
+  background-image: url(../images/text-bg.jpg);
+  background-clip: inherit;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  display: block;
+  background-size: cover;
+  margin-top: 1px;
+  padding-bottom: 20px;
+}
+
+p {
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
+
+.title h2 {
+  color: var(--secondary-white);
+  text-align: center;
+  font-size: 1.6rem;
+  font-weight: 400;
+}
+
+.Hero p {
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  font-size: 0.9rem;
+  font-weight: 600;
+  line-height: 1.6rem;
+  background-color: var(--primary-white);
+}
+
+h3 {
+  margin-block-start: 1.33em;
+  margin-block-end: 1.33em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
+
+p.date {
+  font-weight: 900;
+  font-size: 1.5rem;
+}
+
+h4 {
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
+
+h5 {
+  display: block;
+  font-size: 0.83em;
+  margin-block-start: 1.67em;
+  margin-block-end: 1.67em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  font-weight: bold;
+}
+
+.main-text-color {
+  color: var(--main-color);
+}
+
+.dark-text-color {
+  color: var(--dark-color);
+}
+
+.white-text-color {
+  color: var(--primary-white);
+}
+
+.text-secondary-white {
+  color: var(--secondary-white);
+}
+
+.main-bg-color {
+  background-color: var(--main-color);
+}
+
+.dark-bg-color {
+  background-color: var(--dark-color);
+}
+
+.text-center {
+  text-align: center;
+}
+
+.hide {
+  display: none;
+}
+
+.underline {
+  height: 1px;
+  width: 18%;
+  margin: 1rem auto;
+  background-color: var(--main-color);
+}
+
+.program {
+  display: block;
+}
+
+.program_content {
+  color: #d3d3d3;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 0.5rem;
+  background-image: url(../images/pattern.jpg);
+  background-repeat: repeat;
+}
+
+.program_details {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  width: 100%;
+}
+
+.program_card {
+  display: grid;
+  grid-template-columns: 15% 20% auto;
+  align-items: center;
+  background-color: #ffffff21;
+}
```

Comment:
- [ ] kindly consider giving those cards some padding right to make it more clean and professional
- your design to be fixed 👇
<img width="361" alt="Screen Shot 2023-06-30 at 23 20 09" src="https://github.com/yusufyusie/Africa-Day-2023-Concert/assets/60146030/d33916b5-1210-48eb-a325-abfdc05b5dd4">

-